docs/scenario_editor/scenario_files.md

```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
 
 The AI vehicle behavior details and tags that can be altered to create the scenario file can be found in [Scenario Creation](../scenarios)
```

Comment:
I feel like this file and the **scenarios** should be merged one section called Scenario File Creation (similar to the Trajectory File Creation)  and there should be a sub-section for the  Scenario Editor and a subsection for the Scenario Setup. What do you think? Or at least have these sections one after another on the left bar.
![image](https://user-images.githubusercontent.com/10260895/103847417-cca1b200-5065-11eb-9edc-21a0bcc26fc1.png)